### DISCUSSION TITLE
SyntaxError in examples cms-sanity

### DISCUSSION THREAD

**Discussion by nashyvan** [vercel/next.js#46447] [Help]
https://github.com/vercel/next.js/tree/canary/examples/cms-sanity

I have an error after clicking Deploy button in readme file:
> SyntaxError: Named export 'LIST_NEST_MODE_HTML' not found. The requested module '@portabletext/toolkit' is a CommonJS module, which may not support all module.exports as named exports.

<details>
<summary>Screenshot ✨</summary>
<img width="1438" alt="image" src="https://user-images.githubusercontent.com/44340459/221426506-9bf85ab8-73d0-4935-9429-0d4892a0a62c.png">
</details>

I have the same issue after cloning at localhost:

<details>
<summary>Screenshot ✨</summary>
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/44340459/221427261-7e0f3b5c-1a77-41f8-a995-c4746582b383.png">
</details>

**Comment by TrystanHumann** [comment]
Any solution?

**Comment by Paddy-Hamilton** [comment]
Duplicate of #46450
